content/en/blog/2024/otel-operator-q-and-a/index.md

```diff
@@ -24,7 +24,7 @@ easier. It does the following:
   [custom resource (CR)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/).
 - Manages the configuration of a fleet of OpenTelemetry Collectors via
   [OpAMP](/docs/specs/opamp/) integration, supported by the
-  [`OpAMPBridge`](https://github.com/open-telemetry/opentelemetry-operator/blob/main/docs/api.md#opampbridge)
+  [`OpAMPBridge`](https://github.com/open-telemetry/opentelemetry-operator/blob/main/apis/v1alpha1/opampbridge_types.go)
```

Comment:
Revert the URL change and use `v0.99.0` instead of `main` here and elsewhere in this blog post. For example:

```suggestion
  [`OpAMPBridge`](https://github.com/open-telemetry/opentelemetry-operator/blob/v0.99.0/docs/api.md#opampbridge)
```